Cargo.toml

```diff
@@ -10,13 +10,22 @@ members = [
     "fmt",
 ]
 
-# Binary size optimizations
+[profile.test]
+# Speeds up build
+debug = 0
+# Speeds up tests
+opt-level = 3
+
 [profile.release]
-opt-level = "z"
-lto = true
+# Optimize for binary size, but keep loop vectorization
+opt-level = "s"
```

Comment:
`opt-level = "z"` is the same as `s` except it also disables loop vectorization which we probably want?